packages/manager/apps/container/src/container/legacy/server-sidebar/SidebarItemRenderer.tsx

```diff
@@ -46,104 +62,128 @@ export default function ServerSidebarItemRenderer({
   const { setIsResponsiveSidebarMenuOpen } = useLegacyContainer();
   const hasToggle = item.loader || item.subItems?.length > 0;
   const updateHandler = useCallback(() => onSearch(item), [item, onSearch]);
+  const [isClicked, setIsClicked] = useState(false);
   let itemRender = null;
+
+  let linkColor = OdsThemeColorIntent.text;
+  if (item.isSelected) {
+    linkColor = OdsThemeColorIntent.primary;
+  }
+
+  const handleClick = () => {
+    setIsClicked(!isClicked);
+    onToggle(item);
+  };
   if ('searchField' in item) {
     itemRender = (
       <ServerSidebarSearchField item={item} onUpdate={updateHandler} />
     );
   } else if (item.isNoResultField) {
-    itemRender = (
-      <div>
-        {t('sidebar_no_result')}
-      </div>
-    );
+    itemRender = <div>{t('sidebar_no_result')}</div>;
   } else if (hasToggle) {
-    let icon = 'oui-icon-chevron-right';
+
+    const chevron = item.isOpen ? OdsIconName.CHEVRON_DOWN : OdsIconName.CHEVRON_RIGHT;
+    let iconOrSpinner;
+
     if (item.loadingError) {
-      icon = 'oui-icon-error-circle';
-    } else if (item.isOpen) {
-      icon = 'oui-icon-chevron-down';
+      iconOrSpinner = (
+        <OsdsIcon
+          name={OdsIconName.ERROR_CIRCLE}
+          size={OdsIconSize.xxs}
+          color={OdsThemeColorIntent.text}
+        />
+      );
+    } else if (item.isLoading) {
+      iconOrSpinner = <OsdsSpinner size={OdsSpinnerSize.sm} />;
+    } else {
+      iconOrSpinner = (
+        <OsdsIcon
+          name={chevron}
+          size={OdsIconSize.xxs}
+          color={OdsThemeColorIntent.text}
+        />
+      );
     }
+
     itemRender = (
-      <button
-        type="button"
-        disabled={item.isLoading}
-        onClick={() => onToggle(item)}
-        {...{
-          'aria-busy': !!item.isLoading,
-        }}
-        title={item.isOpen ? '' : t('sidebar_load_services')}
-        className={`${item.depth > 0 ? style.subButton : ''} ${
-          item.isSelected ? style.selectedButton : ''
-        }`}
-      >
-        {item.isLoading && (
-          <span className={style.menuItemIcon} aria-hidden="true">
-            <Spinner
-              as="span"
-              animation="border"
-              size="sm"
-              role="status"
-              aria-hidden="true"
-            />
-          </span>
-        )}
-        {!item.isLoading && (
-          <span className={style.menuItemIcon} aria-hidden="true">
-            <i className={`oui-icon ${icon}`}></i>
-          </span>
-        )}
-        {item.icon && (
-          <span className={style.menuItemIcon} aria-hidden="true">
-            {item.icon}
-          </span>
-        )}
-        <span>{item.label}</span>
+      <button onClick={handleClick} className={style.transparentButton}>
```

Comment:
In the chakra version, I see that `aria-busy` is set when loading. Can we have the same in this version as well?